app/features/atp/information/ui/idx/controller.ts

```diff
@@ -160,7 +185,25 @@ const postInfo = async (
     addToList(req, "surname", {
       surname: req.body["surname"],
     });
-    const allJson = req.session.userData.basicInfo;
+    addToList(req, "addressLine1", {
+      surname: req.body["addressLine1"],
+    });
+    addToList(req, "addressLine2", {
+      surname: req.body["addressLine2"],
+    });
+    addToList(req, "addressTown", {
+      surname: req.body["addressTown"],
+    });
+    addToList(req, "addressCounty", {
+      surname: req.body["addressCounty"],
+    });
+    addToList(req, "addressPostcode", {
+      surname: req.body["addressPostcode"],
+    });
+    function filterInformation(allIdentityEvidence) {
+      return allIdentityEvidence.type == EvidenceType.BASIC_INFO;
+    }
+    const allJson = req.session.sessionData.identityEvidence.filter(filterInformation).slice(-1).map((evidence) => evidence.attributes)[0];
```

Comment:
This could be cleaned up, no need to define a function, can just use a lambda / arrow function